source/SpaceportPanel.cpp

```diff
@@ -48,7 +48,7 @@ SpaceportPanel::SpaceportPanel(PlayerInfo &player)
 
 void SpaceportPanel::UpdateNews()
 {
-	const News *news = GameData::PickNews(player.GetPlanet());
+	const News *news = GameData::PickNews(player.GetPlanet(), player);
```

Comment:
I'm up for moving the PickNews method to SpaceportPanel, and making GameData return the collection of known News. Then we don't need to pass the player object around as much.